src/com/owncloud/android/lib/resources/status/OCCapability.java

```diff
@@ -77,6 +78,7 @@ public OCCapability(){
         mVersionMicro = 0;
         mVersionString = "";
         mVersionString = "";
+        mServerName = "Nextcloud";
```

Comment:
It was not possible to do that as there is no context in that part of the library. I decided to leave the default value empty, that leaves handling up to the application using the library.